data/throughput_benchmarks/workloadc.yaml

```diff
@@ -20,31 +20,31 @@ benchmarks:
       cloud_spanner_config: regional-us-west1
       cloud_spanner_nodes: 3
 
-      # GCE Provisioning: 5 In-Region VMs per Spanner Node
-      ycsb_client_vms: 15
+      # GCE Provisioning: 30 In-Region VMs per Spanner Node
+      ycsb_client_vms: 30
```

Comment:
Given that Cloud Spanner node size will be 3 (L:21), should we either say 10 In-Region VMs per Spanner Node or set the client VM count to 90?

Reply:
Ack